src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx

```diff
@@ -97,6 +97,9 @@ interface PostLogConfig = {
 To enable simulcast for the meeting pass the `simulcastEnabled` prop with value set to `true`. By default, it is `false`.
 For more information on Simulcast, check Amazon Chime JS SDK [simulcast guide](https://github.com/aws/amazon-chime-sdk-js/blob/master/guides/05_Simulcast.md).
 
+#### enableWebAudio
+To enable web audio for the meeting pass the `enableWebAudio` prop with value set to `true`. By default, it is `false`.
```

Comment:
nit: I think a "," needs to be added after "for the meeting".

Reply:
updated